plugin/plugin.go

```diff
@@ -58,24 +58,26 @@ import (
 	"github.com/gogo/protobuf/proto"
 	descriptor "github.com/gogo/protobuf/protoc-gen-gogo/descriptor"
 	"github.com/gogo/protobuf/protoc-gen-gogo/generator"
+	"github.com/gogo/protobuf/vanity"
 	"github.com/mwitkow/go-proto-validators"
 )
 
 func init() {
-	generator.RegisterPlugin(NewPlugin())
+	generator.RegisterPlugin(NewPlugin(true))
```

Comment:
We don't really. It's registering as a plugin with the main gogo generator, but we never even invoke that.